Projects.md

```diff
@@ -0,0 +1,102 @@
+| [lua_sandbox][lsb]                  | Generic sandbox for safe data analysis
+| [lua_sandbox_extensions][lsbx]      | Modules and packages that extend the Lua sandbox
+| [nginx_moz_ingest][nmi]             | Nginx module for Telemetry data ingestion
+| [Generic Ingestion][gi]             | Proposal for streamlined data ingestion of structured data
+| [puppet-config][puppet]             | Cloud services puppet config for deploying infrastructure
+
+[schemas]: https://github.com/mozilla-services/mozilla-pipeline-schemas
+[hs]: https://github.com/mozilla-services/hindsight
+[lsb]: https://github.com/mozilla-services/lua_sandbox
+[lsbx]: https://github.com/mozilla-services/lua_sandbox_extensions
+[nmi]: https://github.com/mozilla-services/nginx_moz_ingest
+[gi]: https://docs.google.com/document/d/1PqiF1rF2fCk_kQuGSwGwildDf4Crg9MJTY44E6N5DSk/edit?ts=5910c4cf#heading=h.74qlucdvwdg0
+[puppet]: https://github.com/mozilla-services/puppet-config/tree/master/pipeline
+
+### EMR Bootstrap scripts
+
+| Name and repo                     | Description                             |
+|-----------------------------------|-----------------------------------------|
+| [emr-bootstrap-spark][eb_spark]   | AWS bootstrap scripts for Spark.
+| [emr-bootstrap-presto][eb_presto] | AWS bootstrap scripts for Presto.
+| [emr-bootstrap-hbase][eb_hbase]   | AWS bootstrap scripts for HBase.
+
+[eb_spark]: https://github.com/mozilla/emr-bootstrap-spark
+[eb_presto]: https://github.com/mozilla/emr-bootstrap-presto
+[eb_hbase]: https://github.com/mozilla/emr-bootstrap-hbase
+
+## Data applications
+
+| Name and repo                   | Description                             |
+|---------------------------------|-----------------------------------------|
+| [telemetry.mozilla.org][tmo_gh] | Main entry point for viewing [aggregate Telemetry data][tmo]
+| [analysis.t.m.o][atmo_gh]       | [Self serve data analysis platform][atmo]
+| [Mission Control][mc]           | Low latency dashboard for stability and health metrics
+| [Experiments Viewer][ev]        | Visualization for [Shield][shield] experiment results
+| [re:dash][redash]               | Mozilla's fork of the [data query / visualization system][stmo]
+| [TAAR][taar]                    | Telemetry-aware addon recommender
+| [Ensemble][ensemble]            | A minimalist platform for publishing data
+
+[tmo_gh]: https://github.com/mozilla/telemetry-dashboard
+[atmo_gh]: https://github.com/mozilla/telemetry-analysis-service
+[mc]: https://github.com/mozilla/missioncontrol
+[ev]: https://github.com/mozilla/experiments-viewer
+[redash]: https://github.com/mozilla/redash
+[taar]: https://github.com/mozilla/taar
+[ensemble]: https://github.com/mozilla/ensemble
+[shield]: https://wiki.mozilla.org/index.php?title=Firefox/Shield
+[tmo]: https://telemetry.mozilla.org
+[atmo]: https://analysis.telemetry.mozilla.org
+[stmo]: https://sql.telemetry.mozilla.org
+
+## Reference materials
+
+### Public
```

Comment:
Should link to [firefox-source-docs telemetry section](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/). Perhaps the [Adding a new telemetry probe](https://developer.mozilla.org/en-US/docs/Mozilla/Performance/Adding_a_new_Telemetry_probe), but I'm not as convinced about the latter.